lib/plugin.c

```diff
@@ -24,6 +24,19 @@
 
 #include <mptcpd/plugin.h>
 
+/**
+ * @todo Remove this preprocessor symbol definition once support for
+ *       path managemnent strategy names are supported in the new
+ *       generic netlink API.
+ *
+ * @note @c GENL_NAMSIZ is used as the size since the path manager
+ *       name attribute in the deprecated MPTCP generic netlink API
+ *       contained a fixed length string of that size.
+ */
+#ifndef MPTCP_PM_NAME_LEN
+# include <linux/genetlink.h>  // For GENL_NAMSIZ
```

Comment:
clang-format doesn't preserve this indentation style, fwiw

Reply:
I couldn't get clang-format to support this indentation style.  As it stands, the mptcpd `.clang-format` file brings code close the mptcpd coding style, but not exactly like it in some cases.